packages/itwin/property-grid/src/components/ElementList.tsx

```diff
@@ -198,13 +128,15 @@ export const ElementList = ({
         </div>
       </div>
       <div className="property-grid-react-element-list-container">
-        {dataProvider && (
-          /* eslint-disable-next-line deprecation/deprecation */
+        {data && (
           <Table
-            dataProvider={dataProvider}
-            onRowsSelected={onRowsSelected}
-            selectionMode={SelectionMode.Single}
-            hideHeader={true}
+            columns={columns}
+            data={data}
+            emptyTableContent="No data."
+            density="extra-condensed"
+            selectionMode="single"
+            onRowClick={onRowClick}
+            // hideHeader={true}
```

Comment:
this might be a missing feature in iTwinUI-react, maybe open an issue in our repo

although i'm not sure we can support this until 2.0 because of https://github.com/iTwin/iTwinUI-react/issues/153

Reply:
@achrysaetos maybe we don't hide the header then for the time being, and maybe set the header to be the element list title, and we dont show it again above the table, just the back & forward buttons?

Reply:
i didnt see a good list component to use in the itwinui lib, lmk what you think would be a better option here

Reply:
If you need a clickable list item, maybe you can use our `<MenuItem>` component. Just make sure to specify [`role='listitem'`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/listitem_role) for accessibility.

Also we encourage you to open issues in our [github repo](https://github.com/iTwin/iTwinUI-react/issues) for any missing features.

Reply:
If you're using `MenuItem` outside a `Menu`, please add the correct roles for correct semantics. The container should have `role='list'` and every item should have `role='listitem'`. See the link in my previous comment for more info.